x-pack/test/functional/page_objects/reporting_page.ts

```diff
@@ -143,20 +138,6 @@ export class ReportingPageObject extends FtrService {
     await this.testSubjects.click('reportModeToggle');
   }
 
```

Comment:
@pheyos I guess we could leave this one here.  What say you Robert?

Reply:
Yes, I think `checkForReportingToasts` should rather stay here as it's reporting specific and not a general toast handling utility.